data-prepper-plugins/geoip-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/GeoIPProcessor.java

```diff
@@ -37,15 +37,13 @@
 public class GeoIPProcessor extends AbstractProcessor<Record<Event>, Record<Event>> {
 
   private static final Logger LOG = LoggerFactory.getLogger(GeoIPProcessor.class);
-  //TODO: rename metrics
-  static final String GEO_IP_PROCESSING_MATCH = "geoIpProcessingMatch";
-  static final String GEO_IP_PROCESSING_MISMATCH = "geoIpProcessingMismatch";
-  private final Counter geoIpProcessingMatchCounter;
-  private final Counter geoIpProcessingMismatchCounter;
+  static final String GEO_IP_EVENTS_PROCESSED = "eventsProcessed";
+  static final String GEO_IP_EVENTS_FAILED_DB_LOOKUP = "eventsFailedDBLookup";
```

Comment:
`eventsFailedDBLookup` the use of the term "DB" implies there is a database. But, we may have other implementations in the future. I think we should rename this to be clearer. Perhaps `eventsFailedLookup` would be sufficient.